build_docs.pl

```diff
@@ -170,7 +170,9 @@ sub _guess_edit_url {
 
     local $ENV{GIT_DIR} = dir($toplevel)->subdir('.git');
     my $remotes = eval { run qw(git remote -v) } || '';
-    if ($remotes !~ m|\s+(\S+[/:]elastic/\S+)|) {
+    if ($remotes !~ m|\s+(\S+[/:]elastic(?:search-cn)?/\S+)|) {
+        # We need either an elastic or and elasticsearch-cn organization. All
```

Comment:
My eyes :heart: your delimiters.

Typo in comment: "an elastic or **and**". Maybe worth sneaking a fix into an unrelated patch, later.

Reply:
:heart:!

In this repo I can fix it in this PR. Much easier!